### PR TITLE
South Zone area Overhaul!

### PR DESCRIPTION
**Pull request by Tokorizo**
## About The Pull Request

PR of https://github.com/DesertRose2/desertrose/pull/548

## Why It's Good For The Game

The current area is meh, it's a mix of three things, the Vault City map. The old BD area, and the mining dungeon from DR. Honestly it really looks cluttered and I personally think it's ugly. This adds new fresh content and also a new mini-dungeon to the surface as well as new places to fight and loot for. It also keeps the old ghoul vault ladder at the end for anyone to use.
## Changelog
:cl:
add: Added new area
/:cl:

### CODE REVIEW

**Comment by BadIsMyUsername** [comment]
What's up with this area? Five nightstalkers that do a ton of poison and brute damage, instantly rush you and surround you so you cannot run away, and you cannot push past/swap positions unlike other mobs. There's no warning to how dangerous the door is, and it's placed in an area with at most ghouls and robots. I could understand if it was placed below ground/in a dungeon area, but fuck me. I cannot figure out a way that I could have survived this without heavy arms and metaknowledge of what the door already contained behind it. 

tl;dr - Yes, I am incredibly salty, you can trash this now.
Wow thanks
